### PR TITLE
EventPipeProfiler documentation

### PR DESCRIPTION
**Pull request by WojciechNagorski**
fixes for #1416

I've published my post here: https://wojciechnagorski.com/2020/04/cross-platform-profiling-.net-code-with-benchmarkdotnet/
I haven't had too much time, but done is better than perfect ;) 

Please let me know if everything is ok. if it is ok, then I will publish information about my post on twitter

### CODE REVIEW

**Comment by AndreyAkinshin** [comment]
It looks great, thanks! I'm going to reference your blog post from the release notes.

Could you please move the image to an external place? For example, you can create a new gist, add the image as a comment, and reuse the image url in the documentation.
I'm trying to avoid holding binary images in the repo (it helps to keep the `.git` folder small).

**Comment by AndreyAkinshin** [comment]
@WojciechNagorski also, please add a link to SpeedScope in `IntroEventPipeProfiler.md` and `IntroEventPipeProfilerAdvanced.md`.

**Comment by WojciechNagorski** [comment]
Ok. No problem. Can I do it tomorrow?

**Comment by WojciechNagorski** [comment]
@AndreyAkinshin I've fixed review.

**Comment by AndreyAkinshin** [comment]
@WojciechNagorski thanks!

File: docs/articles/features/event-pipe-profiler.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿---
+uid: docs.event-pipe-profiler
+name: EventPipeProfiler
+---
+
+# EventPipeProfiler
+
+`EventPipeProfiler` is a cross-platform profiler that allows profile .NET code on every platform - Windows, Linux, macOS. Collected data are exported to trace files (`.speedscope.json` and `.nettrace`) which can be analyzed using [SpeedScope](https://www.speedscope.app/), [PerfView](https://github.com/Microsoft/perfview), and [Windows Performance Analyzer](https://docs.microsoft.com/en-us/windows-hardware/test/wpt/windows-performance-analyzer). This new profiler is available from the 0.12.1 version.
```
Comment (adamsitnik) [inline docs/articles/features/event-pipe-profiler.md:8]: I know that `Visual Studio Profiler` supports it, I am not sure about  `Windows Performance Analyzer`

File: docs/articles/samples/IntroEventPipeProfilerAdvanced.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿---
+uid: BenchmarkDotNet.Samples.IntroEventPipeProfilerAdvanced
+---
+
+## Sample: EventPipeProfilerAdvanced
+
+The most advanced and powerful way to use `EventPipeProfiler` is a custom configuration. As you can see the below configuration adds `EventPipeProfiler` that constructor can take the profile and/or a list of providers. 
+Both `EventPipeProfiler` and `dotnet trace` use the `Microsoft.Diagnostics.NETCore.Client` package internally. So before you start using the custom configuration of this profiler, it is worth reading the documentation [here](https://github.com/dotnet/diagnostics/blob/master/documentation/dotnet-trace-instructions.md) and [here](https://docs.microsoft.com/en-us/dotnet/core/diagnostics/dotnet-trace#dotnet-trace-collect) where you can fine more information about how to configure provider list.
```
Comment (adamsitnik) [inline docs/articles/samples/IntroEventPipeProfilerAdvanced.md:8]: nit: typo fin**e** -> fin**d**

**Comment by WojciechNagorski** [comment]
@adamsitnik Thanks! I'll fix it.

**Pull request merged**